documentation/tutorials/check.md

```diff
@@ -0,0 +1,174 @@
+
+<p align="center"><img src="/assets/tutorial/check/change-report-example.png" style="max-width: 100%"/></p>
+
+## Applying the changes
+
+We now know that applying our pending migration(s) will have the intended effect on our target schema, so we can go ahead and apply them using `migrate`:
+
+<pre class="console">./flyway migrate</pre>
+
+## Detecting drift using `-drift` flag
+
+Before applying future migrations, we might also want to confirm that no drift has occured in the target database, i.e. all schema changes have been applied in the form of flyway migrations, as oppose to unrecorded manual changes. 
+
+To demostrate this, we can apply a manual change to the target database, e.g. using SSMS:
+
+```sql
+use foobar;
+
+alter table PERSON add EMAIL varchar(100);
+```
+
+We can then run `check` with the `-drift` flag:
+
+<pre class="console">./flyway check -drift</pre>
+
+This creates another pair of HTML and JSON reports. This time the files contain a drift report, showing how the target database schema (in this case `foobar`) differs from whats contained in the applied migrations. 
+
+## Using Check in Docker
+
+This feature can also be accessed through the Flyway Enterprise Docker images.
+
+First pull the relevant images for this tutorial:
+
+<pre class="console">docker pull mcr.microsoft.com/mssql/server</pre>
+<pre class="console">docker pull redgate/flyway</pre>
+
+We can then use Docker to set up an instance of SQL Server with a login user `sa` and password `Flyway123`:
+
+<pre class="console">docker run -e "ACCEPT_EULA=Y" -e "SA_PASSWORD=Flyway123" -p 1433:1433 -d mcr.microsoft.com/mssql/server</pre>
+
+*Make sure there are no other SQL Server instances running with the same port*
+
+As before, create a target database `foobar` and a temporary database `check_temp_db` e.g. using SSMS.
+
+Create a folder called `sql` to contain migrations and add a new SQL migration `V1__add_table.sql`:
+
+```sql
+create table PERSON (
+    ID int not null,
+    NAME varchar(100) not null
+);
+```
+
+Along with `sql`, you might also want to create a `reports` folder to store the reports that will be produced, and a `conf` folder to store the configuration file.
+
+Inside `conf`, place a file called `flyway.conf` and add the following contents:
+
+```
+flyway.url=jdbc:sqlserver://localhost;databaseName=foobar;trustServerCertificate=true
+flyway.user=sa
+flyway.password=Flyway123
+flyway.licenseKey=<put your license key here>
+
+flyway.check.tempUrl=jdbc:sqlserver://localhost;databaseName=check_temp_db;trustServerCertificate=true
+flyway.check.outputLocation=reports
```

Comment:
As with a previous comment, this parameter is now gone so the tutorial will need to be updated to reflect that